module/VuFindTheme/src/VuFindTheme/InjectTemplateListenerFactory.php

```diff
@@ -90,6 +92,54 @@ function ($prefix) use ($exclude) {
             }
         );
 
+        $this->loadConfiguredJavascriptFilesFromMixin($config, $container);
```

Comment:
Why is this happening in InjectTemplateListenerFactory? This doesn't seem like the right place -- I'm not sure why Javascript loading should be configured as a side effect of constructing the InjectTemplateListener. Would it make more sense for this to happen in the VuFindTheme\Initializer class? Should the VuFindTheme\ThemeInfo class be involved somehow to keep abstraction of mixins in one place (and/or pre-load all the mixin configs once so we don't have to load them again to process this part)?